packages/documentation/gatsby-config.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     title: `VA.gov Developer's Documentations`,
     description: `Resources and documention for the Development within the VA.gov project`,
   },
+  pathPrefix: '/veteran-facing-services-tools',
```

Comment:
This works- there are some fonts failing to load but I think we can address that separately.